src/main/java/frc/robot/subsystems/grabber/Grabber.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.subsystems.grabber;
+
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.util.function.DoubleSupplier;
+
+@Logged
+public class Grabber extends SubsystemBase {
+    private final double VOLTAGE_THRESHOLD = 1.0;
+
+    private final GrabberIO io;
+    private final GrabberInputs inputs = new GrabberInputs();
+
+    private double targetVoltage = 0.0;
+
+    public Grabber(GrabberIO io) {
+        this.io = io;
+    }
+
+    @Override
+    public void periodic() {
+        io.updateInputs(inputs);
+    }
+
+    public double getCurrentVelocity() {
+        return inputs.currentVelocity;
+    }
+
+    public double getCurrentVoltage() {
+        return inputs.currentAppliedVoltage;
+    }
+
+    public double getCurrentPosition() {
+        return inputs.currentPosition;
+    }
+
+    public boolean atTargetVoltage() {
```

Comment:
I don't think having an `atTargetVoltage` method is necessary. The voltage is applied almost instantly to the motor. The only reason the applied voltage will not be the target voltage is when the motor is current limited.